assets/js/components/ChargingSessionModal.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+	<Teleport to="body">
+		<div
+			id="sessionDetailsModal"
+			class="modal fade text-dark"
+			tabindex="-1"
+			role="dialog"
+			aria-hidden="true"
+		>
+			<div class="modal-dialog modal-dialog-centered modal-dialog-scrollable" role="document">
+				<div v-if="session != undefined" class="modal-content">
+					<div class="modal-header">
+						<h5>{{ $t("session.title") }}</h5>
+						<button
+							type="button"
+							class="btn-close"
+							data-bs-dismiss="modal"
+							aria-label="Close"
+						></button>
+					</div>
+					<div class="modal-body">
+						<table class="table">
+							<tbody>
+								<tr>
+									<th colspan="2"></th>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("sessions.csv.created") }}
+									</th>
+									<td>
+										{{ fmtFullDateTime(new Date(session.created), false) }}
+									</td>
+								</tr>
+								<tr>
+									<th>
+										{{ $t("sessions.csv.finished") }}
```

Comment:
We shouldn't used the `.csv.` texts in the web UI. I'd suggest to stick to the "Period" headline we use in the table and put start/end-time in two separate lines of the same cell.

Reply:
I can understand that. Then I'd suggest to create two new label for the ui "session.started", "session.finished". "Created" is quite technical. Maybe we should also change this in the csv header.